### PR TITLE
 Modifications to daemon.py admin_config.py installer.py

### PR DESCRIPTION
**Pull request by rupamk**
Included Master Slave optional configuration to enable/disable rubix.cluster.on-master=true flag.

### CODE REVIEW

File: rubix_admin/admin_config.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (vrajat) [inline rubix_admin/admin_config.py]: Please remove the whitespace diffs

File: rubix_admin/daemon.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (vrajat) [inline rubix_admin/daemon.py]: Same here. Is this an editor setting in your IDE ?

Comment (rupamk) [reply rubix_admin/daemon.py]: Maybe. I will fix that. Thanks.

**Comment by vrajat** [comment]
Changes look good to me. Can you please comment on how you tested this change ? Also there is a swp file in the diff which shouldn't be there.

**Comment by rupamk** [comment]
Tested with the latest code

Coordinator Log
`18/08/06 05:13:00,637 DEBUG pool-7-thread-1 bookkeeper.CoordinatorBookKeeper: Received heartbeat from ip-172-31-39-130.us-east-2.compute.internal`

Worker Log:
`18/08/06 05:14:00,630 DEBUG HeartbeatService RUNNING bookkeeper.WorkerBookKeeper: Sending heartbeat to 172.31.35.214`

**Pull request merged**